tutorial/tutorial-files/Coin.sol

```diff
@@ -0,0 +1,67 @@
+
+// Specifies that the source code is for a version
+// of Solidity greater than 0.5.0
+pragma solidity >=0.5.0 <0.7.0;
```

Comment:
why not just 0.7.0?

Reply:
From the Ethereum Studio example. Good to define a minimum since the code wouldn't compile in lower versions.